<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FenKey</class>
 <widget class="QDialog" name="FenKey">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>626</width>
    <height>218</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Enter secret key</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_6">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:11pt; font-weight:600; color:#ff0000;&quot;&gt;Do not forget your key !!! It is impossible to decrypt a file without the key&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="textFormat">
      <enum>Qt::RichText</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="4" column="0" colspan="3">
       <widget class="QLineEdit" name="EditConfirm"/>
      </item>
      <item row="2" column="0" colspan="3">
       <widget class="QLineEdit" name="EditPass"/>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Passphrase</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Confirmation</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_2">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QComboBox" name="comboAlgo">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
        <item>
         <property name="text">
          <string>Serpent/GCM</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Threefish-512/EAX</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>AES-256/CTR-BE</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkDelete">
        <property name="text">
         <string>Delete original</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="ButtonCancel">
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="ButtonOK">
        <property name="text">
         <string>Ok</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>ButtonCancel</sender>
   <signal>clicked()</signal>
   <receiver>FenKey</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>474</x>
     <y>231</y>
    </hint>
    <hint type="destinationlabel">
     <x>313</x>
     <y>134</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
